examples/post-processing/rs-post-processing.cpp

```diff
@@ -0,0 +1,411 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2017 Intel Corporation. All Rights Reserved.
+
+#include <librealsense2/rs.hpp> // Include RealSense Cross Platform API
+#include "example.hpp"          // Include short list of convenience functions for rendering
+
+#include <imgui.h>
+#include "imgui_impl_glfw.h"
+
+// a time interval which must pass between movement of the pointcloud
+#define ROTATION_DELTA 1e7
+
+// maximum angle for the rotation of the pointcloud
+#define MAX_ANGLE 90
+
+// for the interface: distance between checkbox and slider
+#define DIST 120
```

Comment:
Please prefer constant variables over preprocessor definitions.
Also - if it is not used everywhere in the file, declare it when you need it. (In this case inside the `render_ui` which it the only place you use it)

Reply:
Same comment for the rest of the defines here.